src/view/components/helpText.styl

```diff
@@ -1,3 +1,9 @@
 .spectrum-semantic-negative-color-text-small {
   color: var(--spectrum-semantic-negative-color-text-small);
 }
+
+.warning-container-content {
+  svg {
```

Comment:
Not a big deal, but rather than selecting all `svg` tags inside the container, we might want to be more specific. Maybe a selector like `[arria-label="Alert"]` would be slightly better?

Reply:
Noted. Thanks for the review!